core/models.py

```diff
@@ -27,11 +29,27 @@
 from wagtail.utils.decorators import cached_classmethod
 from wagtail_personalisation.blocks import PersonalisedStructBlock
 from wagtail_personalisation.models import PersonalisablePageMixin
+from wagtailmedia.models import Media
 
 from core import blocks as core_blocks, mixins
 from core.context import get_context_provider
 
 
+class GreatMedia(Media):
+
+    transcript = models.TextField(verbose_name=_('transcript'), blank=True, null=True)
```

Comment:
Tiny thing, missed it first time around.

```suggestion
    transcript = models.TextField(verbose_name=_('Transcript'), blank=True, null=True)
```

If you change it here, edit the migration too so that you don't have to regenerate another one